crypto/lhash/lh_stats.c

```diff
@@ -20,7 +20,14 @@
 #include <openssl/lhash.h>
 #include "lhash_local.h"
 
+# ifndef OPENSSL_NO_DEPRECATED_3_1
```

Comment:
Defining `OPENSSL_SUPPRESS_DEPRECATED` is the usual way we use / define deprecated function.

Reply:
Done.